src/core/lib/security/security_connector/load_system_roots_supported.cc

```diff
@@ -63,14 +63,10 @@ const char* kCertDirectories[] = {""};
 #endif                      // GPR_APPLE
 
 grpc_slice GetSystemRootCerts() {
-  grpc_slice valid_bundle_slice = grpc_empty_slice();
   size_t num_cert_files_ = GPR_ARRAY_SIZE(kCertFiles);
   for (size_t i = 0; i < num_cert_files_; i++) {
-    grpc_error_handle error =
-        grpc_load_file(kCertFiles[i], 1, &valid_bundle_slice);
-    if (error.ok()) {
-      return valid_bundle_slice;
-    }
+    auto slice = LoadFile(kCertFiles[i], /*add_null_terminator=*/true);
```

Comment:
nit: This creates an std::string to use for the const string ref, right? `const char* kCertFiles[]`. We could avoid this using string_view.

Reply:
See my response above.  We cannot actually avoid making a copy in this case and still have a reasonable API for `LoadFile()`.